contracts/Company/PlaceholderEquityTokenController.sol

```diff
@@ -243,20 +243,12 @@ contract PlaceholderEquityTokenController is
     // Implements ITokenController
     //
 
-    function onTransfer(address from, address ,uint256)
+    function onTransfer(address broker, address from, address /*to*/, uint256 /*amount*/)
         public
         constant
         returns (bool allow)
     {
-        return _transfersEnabled || from == _commitment;
-    }
-
-    function onTransferFrom(address, address, address, uint256)
-        public
-        constant
-        returns (bool allow)
-    {
-        return _transfersEnabled;
+        return _transfersEnabled || (from == _commitment && broker == from);
```

Comment:
In general a short comment on what this should achieve is not so bad. To me it's clear that only the commitment should be able to make transfers if they're not globally enabled, but it just will help reading

Reply:
OK